Source/WebGPU/WebGPU/Pipeline.mm

```diff
@@ -88,7 +86,6 @@
         const auto& entry = constants[i];
         auto keyEntry = fromAPI(entry.key);
         auto indexIterator = entryPointInformation.specializationConstants.find(keyEntry);
-        // FIXME: Remove code inside the following conditional statement when https://bugs.webkit.org/show_bug.cgi?id=266774 is completed
         if (indexIterator == entryPointInformation.specializationConstants.end()) {
```

Comment:
I think we can remove lines 89..100 as well, this was to workaround the prior lack of unicode support but is no longer needed

Reply:
Well to be more specific, it should be replaced with (from lines 99-100):
```
if (indexIterator == entryPointInformation.specializationConstants.end())
            return { };
```